prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -22,7 +22,7 @@ identifier_2a:
   expressionType: resource
   vars:
     valueIn: ORC.3.1
-    systemCX: ORC.3.2
+    namespaceId: String, OBR.3.2
```

Comment:
Ah good catch, I think it was a bad copy/paste. It seems generally assumed that ORC.3 and OBR.3 will both have the same values, but we should at least pull from the same segment within identifier_2a and identifier_2b at least. Will make that fix.

Reply:
Do we need a TODO to review the mapping combinations for Filler and Placer order numbers as there's potential to overwrite data?

Reply:
@brick-green That potential would just be in cases where OBR and ORC contradict each other? I think we meant to bring that up on standup and/or eng sync, and we probably still should, but my take is that if OBR and ORC contradict each other, that's a validation issue. If we take filler as an example, we're going from 2 values (OBR and ORC) in HL7, down to 1 value in the DiagnosticReport in FHIR, back to 2 values in the HL7-out, so I don't see a mapping solution that would allow for us to have 2 different values in the output without doing some odd custom extension (which would only be useful for what seems to me to be invalid input).

Reply:
yeah, that's the only case. I wonder if an OBR and ORC would be grouped in the same order_observation if the values do not match...I'll test that